downstream/modules/platform/ref-eda-controller-variables.adoc

```diff
@@ -40,6 +40,9 @@ Default = empty list
 | *`automationedacontroller_gunicorn_workers`* | Number of workers for the API served through gunicorn.
 
 Default = (# of cores or threads) * 2 + 1
+| *`automationedacontroller_max_running_activations`* | The number of running activations per worker.
+
+Default = 12
```

Comment:
@jamesmarshall24 - the `automationedacontroller_max_running_activations` variable has been added to Appendix A.5. in the install guide.